polaris-react/src/components/EmptyState/EmptyState.scss

```diff
@@ -127,7 +126,6 @@ $centered-illustration-width: 226px;
 }
 
 .Content {
-  @include text-style-body;
   @include text-emphasis-subdued;
```

Comment:
This should be removed and `color="subdued"` passed in to the `<Text>` component